src/amo/components/Addon/index.js

```diff
@@ -173,114 +185,174 @@ export class AddonBase extends React.Component {
-            />
+    const summaryProps = {};
+    let summaryPlaceholder = null;
+    if (addon) {
+      // Themes lack a summary so we do the inverse :-/
+      // TODO: We should file an API bug about this...
+      const summary = addon.summary ? addon.summary : addon.description;
+      summaryProps.dangerouslySetInnerHTML = sanitizeHTML(summary, ['a']);
+    } else {
+      summaryPlaceholder = <LoadingText />;
+    }
 
-            <InstallButton
-              {...this.props}
-              className="Button--action Button--small"
-              disabled={!compatible}
-              ref={(ref) => { this.installButton = ref; }}
-              status={installStatus}
-            />
+    const titleProps = {};
+    let titlePlaceholder = null;
+    if (addon) {
+      const title = i18n.sprintf(
+        // L10n: Example: The Add-On <span>by The Author</span>
+        i18n.gettext('%(addonName)s %(startSpan)sby %(authorList)s%(endSpan)s'), {
+          addonName: addon.name,
+          authorList: authorList.join(', '),
+          startSpan: '<span class="Addon-author">',
+          endSpan: '</span>',
+        }
+      );
+      titleProps.dangerouslySetInnerHTML = sanitizeHTML(title, ['a', 'span']);
+    } else {
+      titlePlaceholder = <LoadingText />;
+    }
 
-            {this.headerImage({ compatible })}
+    const addonPreviews = addon ? addon.previews : [];
 
-            <h2 className="visually-hidden">
-              {i18n.gettext('Extension Metadata')}
-            </h2>
+    let isCompatible = false;
+    let compatibility;
+    if (addon) {
+      compatibility = getClientCompatibility({
+        addon, clientApp, userAgentInfo,
+      });
+      isCompatible = compatibility.compatible;
+    }
 
-            <AddonMeta addon={addon} />
+    // eslint-disable react/no-danger
+    return (
+      <div className={classNames('Addon', `Addon-${addonType}`)}>
+        {errorBanner}
+        <Card className="" photonStyle>
+          <header className="Addon-header">
+            <section className="Addon-title-section">
+              <h1 className="Addon-title" {...titleProps}>
+                {titlePlaceholder}
+              </h1>
+              <p className="Addon-summary" {...summaryProps}>
+                {summaryPlaceholder}
+              </p>
+              {addon ?
+                <InstallButton
+                  {...this.props}
+                  className="Button--action Button--small"
+                  disabled={!isCompatible}
+                  ref={(ref) => { this.installButton = ref; }}
+                  status={installStatus}
+                /> : null
+              }
+            </section>
+
+            <section className="Addon-metadata">
+              {this.headerImage({ compatible: isCompatible })}
+
+              <h2 className="visually-hidden">
+                {i18n.gettext('Extension Metadata')}
+              </h2>
+              {addon ? <AddonMeta addon={addon} /> : <LoadingText />}
```

Comment:
I think this would benefit from `AddonMeta` being able to handle an empty `addon` and rendering each section as `<LoadingText />` instead of just one bar, eg:

```js
if (!addon) {
      return (
        <div className="AddonMeta">
          <div className="AddonMeta-item AddonMeta-users">
            <h3 className="visually-hidden"><LoadingText /></h3>
            <p className="AddonMeta-text"><LoadingText /></p>
            <p className="AddonMeta-text AddonMeta-review-count">
              <LoadingText />
            </p>
            <Rating className="AddonMeta-Rating" rating={0} readOnly
              styleName="small" />
          </div>
        </div>
      );
    }
```

Though probably better implemented in the component like you did in `Addon` so we aren't duplicating the HTML.